docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+### UID2 Operator error codes
+
+The following table lists possible errors while setting up private operator.
+
+:::note
+This applies only to Private Operators starting from Q2 2024.
+:::
+
+| Error Code | Issue | Steps to Resolve |
+| :--- | :--- | :--- |
+| AW01 | MissingInstanceProfile |  Attach an IAM instance profile to the EC2 instance with the required permissions. The UID2 Operator needs these permissions to access configurations from AWS Secrets Manager. |
+| AW02 | ConfigNotFound | Ensure the secret referenced by the operator exists in AWS Secrets Manager in the same region as the operator. And verify that the IAM instance profile has permission to access it. Check the logs the specific secret name and region. |
+| AW03 | MissingConfig | Required attributes are missing in the configuration. Refer to the logs for details and update the missing attributes in Secrets Manager. |
+| AW04 | InvalidConfigValue | A configuration value is invalid. Verify the configuration values in Secrets Manager align with the required format and environment. Note `debug = true` is allowed only in the `integ` environment. Check the logs for mode details. |
```

Comment:
| AW04 | InvalidConfigValue | A configuration value is invalid. Verify that the configuration values in AWS Secrets Manager align with the required format and environment. Note `debug = true` is allowed only in the `integ` environment. Check the logs for mode details. |

(Verify > Verify that) also suggest we use AWS Secrets Manager throughout since it's their product name.